src/Mobs/UtilityMonster.cpp

```diff
@@ -0,0 +1,44 @@
+
+#include "Globals.h"  // NOTE: MSVC stupidness requires this to be the same across all modules
+
+#include "UtilityMonster.h"
+#include "LineBlockTracer.h"
+#include "../World.h"
+
+
+
+
+
+cUtilityMonster::cUtilityMonster(const AString & a_ConfigName, eMonsterType a_MobType, const AString & a_SoundHurt, const AString & a_SoundDeath, const AString & a_SoundAmbient, float a_Width, float a_Height) :
+	Super(a_ConfigName, a_MobType, a_SoundHurt, a_SoundDeath, a_SoundAmbient, a_Width, a_Height)
+{
+	m_EMPersonality = PASSIVE;
```

Comment:
that is a detail but, isn't it redundant ? ^^

Reply:
Maybe it is, I've basically copied this stuff from `PassiveAggressiveMonster.h'. I'll check

Reply:
I'm not even sure whether making `cUtilityMonster` will be the best course of action. It ensures that `CheckEventSeeMob` or `CheckEventLostMob` is called every tick, but it looks like most hostile mobs will also have to perform these check anyway - they're gonna be hostile towards iron golems.

Unless this is solved in other way, e.g. by making the iron golem set the target of hostile mobs nearby to itself. But that would ignore their range of sight, so... not the best option, I guess